lib/ext/tags.js

```diff
@@ -0,0 +1,96 @@
+'use strict';
+
+// Copyright (c) 2016 Resonance Labs, Inc
```

Comment:
Good catch. Just added a commit to remove it altogether. Nowhere else in the library do we include a copyright notice within the source, so it seemed both simpler and more consistent to remove it.